lib/mongo/lib/compensateForLatency.js

```diff
@@ -23,5 +23,12 @@ export default (channels, event, uid, doc, fields) => {
                 // }
             })
         }
+
+        let directSubs = RedisSubscriptionManager.store[channel + '::' + doc._id];
+        if (directSubs && directSubs.length) {
+            directSubs.forEach(subscriber => {
```

Comment:
don't forget to put channelLastIds until you find a better way for optimistic ui